<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>664</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>211</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string>文件列表：</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="fileListWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>601</width>
     <height>391</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="title">
    <string>预览窗口：</string>
   </property>
   <widget class="QLabel" name="imglabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QVideoWidget" name="videoWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>361</height>
     </rect>
    </property>
    <zorder>groupBox_4</zorder>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>450</y>
     <width>151</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>本机设置：</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>本 机IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="localIP">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>本机端口:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="localPORT">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8888</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLineEdit" name="delayLineEdit"/>
      <widget class="QCheckBox" name="loopCheckBox">
       <property name="text">
        <string>循环</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>重启电脑</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>关机</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>450</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>数据收发：</string>
   </property>
   <widget class="QTextEdit" name="recvTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>22</y>
      <width>421</width>
      <height>94</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QTextEdit" name="sendTextEdit"/>
    <widget class="QPushButton" name="btnSendData">
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>122</y>
      <width>421</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>目标主机:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="targetIP"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>端口:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="targetPORT"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="selectFiles">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>选择路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fileDirLineEdit">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>620</y>
     <width>145</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="videoSlider">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="audioSlider">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>420</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>420</y>
     <width>451</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="videoPlayerPause">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="lastContent">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="videoRewind">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="videoAdvance">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="nextContent">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="videoStop">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="videoMute">
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="position">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>400</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>时长</string>
   </property>
   <property name="statusTip">
    <string>总时间</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fileListWidget</tabstop>
  <tabstop>fileDirLineEdit</tabstop>
  <tabstop>selectFiles</tabstop>
  <tabstop>videoSlider</tabstop>
  <tabstop>position</tabstop>
  <tabstop>videoPlayerPause</tabstop>
  <tabstop>lastContent</tabstop>
  <tabstop>videoRewind</tabstop>
  <tabstop>videoAdvance</tabstop>
  <tabstop>nextContent</tabstop>
  <tabstop>videoStop</tabstop>
  <tabstop>videoMute</tabstop>
  <tabstop>audioSlider</tabstop>
  <tabstop>recvTextEdit</tabstop>
  <tabstop>targetIP</tabstop>
  <tabstop>targetPORT</tabstop>
  <tabstop>sendTextEdit</tabstop>
  <tabstop>btnSendData</tabstop>
  <tabstop>localIP</tabstop>
  <tabstop>localPORT</tabstop>
  <tabstop>delayLineEdit</tabstop>
  <tabstop>loopCheckBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
